src/platform/packages/shared/presentation/presentation_publishing/interfaces/has_edit_capabilities.ts

```diff
@@ -19,6 +19,10 @@ export interface HasEditCapabilities extends HasTypeDisplayName {
   onEdit: () => Promise<void>;
   isEditingEnabled: () => boolean;
   getEditHref?: () => Promise<string | undefined>;
+  getEditPanel?: (options?: {
```

Comment:
Since `getEditPanel` is only used by lens, how about moving its definition to LensApi?